helpers/tools.ts

```diff
@@ -139,7 +139,7 @@ For better results, you can specify the region parameter to get results from a s
     dependencies: [
       {
         name: "e2b_code_interpreter",
-        version: "^0.0.11b38",
+        version: "0.0.11b38",
```

Comment:
new e2b_code_interpreter package is not working with the current code right now. I think better to keep this version and update the code later once we got a official 0.0.11 version.